changelog/undistributed/changelog_show_bgp_vrf_all_all_summary_nxos_20220628.rst

```diff
@@ -0,0 +1,7 @@
+--------------------------------------------------------------------------------
+                            Fix
+--------------------------------------------------------------------------------
+* NXOS
+    * Modified ShowBgpVrfAllAllSummary:
+        * Added regex pattern <p8_3> and <p8_4> to accommodate output of neighbors with 4byte asn.
+
```

Comment:
Can you please edit the timestamp in filename to changelog_show_bgp_vrf_all_all_summary_nxos_%Y%M%D%H%M%S.rst

Reply:
Done.  Pushed another commit with file renamed.